Makefile

```diff
@@ -1,5 +1,5 @@
 PROJECT = mysql_poolboy
 DEPS = mysql poolboy
-dep_mysql = git https://github.com/mysql-otp/mysql-otp.git 0.8.0
+dep_mysql = git https://github.com/mysql-otp/mysql-otp.git 1.0.0
 dep_poolboy = git https://github.com/devinus/poolboy.git 1.4.2
```

Comment:
I'm looking into a way to create a rebar file from the Makefile instead of trying to support 2 files for deps.

My mistake for merging it into quickly.